packages/cli/src/commands/deploy.ts

```diff
@@ -5,18 +5,42 @@ import { BoosterConfig, Logger } from '@boostercloud/framework-types'
 import { Script } from '../common/script'
 import Brand from '../common/brand'
 import { logger } from '../services/logger'
+import { exec } from 'child-process-promise'
+import { wrapExecError } from '../common/errors'
+import * as path from 'path'
 import { currentEnvironment, initializeEnvironment } from '../common/environment'
 
+async function pruneDependencies(config: BoosterConfig): Promise<void> {
+  try {
+    await exec('npm install --production', { cwd: projectDir(config) })
+  } catch (e) {
+    throw wrapExecError(e, 'Could not prune dev dependencies')
+  }
+}
+
+async function reinstallDependencies(skipRestoreDependencies: boolean, config: BoosterConfig): Promise<void> {
+  try {
+    if(skipRestoreDependencies === false){
+      await exec('npm install', { cwd: projectDir(config) })
+    }
+  } catch (e) {
+    throw wrapExecError(e, 'Could not reinstall dependencies')
+  }
+}
```

Comment:
I think that it would make sense to move functions like these to a separate `../common/dependencies.ts` file